products/motorola.md

```diff
@@ -0,0 +1,366 @@
+    eol: 2023-02-01
+    link: https://www.motorola.com/we/smartphones-moto-g-30/p
+-   releaseCycle: "moto-g22"
+    releaseLabel: "moto g22"
+    releaseDate: 2022-03-01
+    eol: 2025-03-01
+    link: https://www.motorola.com/we/smartphones-moto-g-22/p
+-   releaseCycle: "moto-g20"
+    releaseLabel: "moto g20"
+    releaseDate: 2021-05-01
+    eol: 2023-05-01
+    link: https://www.motorola.com/we/smartphones-moto-g-20/p
+-   releaseCycle: "moto-g10-power"
+    releaseLabel: "moto g10 power"
+    releaseDate: 2021-03-01
+    eol: 2023-03-01
+    link: https://www.motorola.in/smartphones-moto-g-10-power/p
+-   releaseCycle: "moto-g10"
+    releaseLabel: "moto g10"
+    releaseDate: 2021-03-01
+    eol: 2023-03-01
+    link: https://www.motorola.com/we/smartphones-moto-g-10/p
+-   releaseCycle: "moto-g-stylus-2021"
+    releaseLabel: "moto g stylus (2021)"
+    releaseDate: 2021-01-01
+    eol: 2023-01-01
+    link: https://www.motorola.com/us/smartphones-moto-g-stylus-gen-2/p
+-   releaseCycle: "moto-g-pure-2021"
+    releaseLabel: "moto g pure (2021)"
+    releaseDate: 2021-11-01
+    eol: 2023-11-01
+    link: https://www.motorola.com/us/smartphones-moto-g-pure/p
+-   releaseCycle: "moto-g-power-2021"
+    releaseLabel: "moto g power (2021)"
+    releaseDate: 2021-01-01
+    eol: 2023-01-01
+    link: https://www.motorola.com/us/smartphones-moto-g-power/p
+-   releaseCycle: "moto-g-play-2021"
+    releaseLabel: "moto g play (2021)"
+    releaseDate: 2021-01-01
+    eol: 2023-01-01
+    link: https://www.motorola.com/we/smartphones-moto-g-play/p
+-   releaseCycle: "moto-g-5g-plus"
+    releaseLabel: "moto g 5g PLUS"
+    releaseDate: 2020-07-01
+    eol: 2022-07-01
+    link: https://www.motorola.com/we/smartphones-moto-g-5g-plus/p
+-   releaseCycle: "moto-g-5g"
+    releaseLabel: "moto g 5g"
+    releaseDate: 2020-11-01
+    eol: 2022-11-01
+    link: https://www.motorola.com/we/smartphones-moto-g-5g/p
+-   releaseCycle: "moto-g9-power"
+    releaseLabel: "moto g9 power"
+    releaseDate: 2020-12-01
+    eol: 2022-12-01
+    link: https://www.motorola.com/we/smartphones-moto-g-power-gen-9/p
+-   releaseCycle: "moto-g9-plus"
+    releaseLabel: "moto g9 plus"
+    releaseDate: 2020-08-01
+    eol: 2022-08-01
+    link: https://www.motorola.com/nz/smartphones-moto-g-plus-gen-9/p
+-   releaseCycle: "moto-g9-play"
+    releaseLabel: "moto g9 play"
+    releaseDate: 2020-08-01
+    eol: 2022-08-01
+    link: https://www.motorola.com/we/smartphones-moto-g-play-gen-9/p
+-   releaseCycle: "moto-g9"
+    releaseLabel: "moto g9"
+    releaseDate: 2020-08-01
+    eol: 2022-08-01
+-   releaseCycle: "moto-g-fast"
+    releaseLabel: "moto g fast"
+    releaseDate: 2020-06-01
+    eol: 2022-06-01
+    link: https://www.motorola.com/us/smartphones-moto-g-fast/p
+-   releaseCycle: "moto-g-stylus"
+    releaseLabel: "moto g stylus"
+    releaseDate: 2020-04-01
+    eol: 2022-04-01
+    link: https://www.motorola.com/us/smartphones-moto-g-stylus/p
```

Comment:
Redirected to https://www.motorola.com/us/smartphones-moto-g-stylus-gen-2/p?skuId=534, is it OK ?